decisiontree/models.py

```diff
@@ -198,6 +199,27 @@ def __str__(self):
         return u"%s : %s --> %s" % (self.current_state, self.answer, self.next_state)
 
 
+class SessionQuerySet(models.query.QuerySet):
+
+    def open(self):
+        return self.exclude(Q(state=None) | Q(canceled=True))
```

Comment:
Yes. However the corresponding query, `closed()`, needs it and I think that using the same conditions for both `open()` and `closed()` makes the code more clear. (I refactored as well to use the conditions stored as a variable.) Is there a reason, such as performance, that we should avoid `Q` objects?